boards/nxp/frdm_mcxn947/board.c

```diff
@@ -18,14 +18,16 @@
 #define BOARD_USB_PHY_TXCAL45DM (0x07U)
 
 usb_phy_config_struct_t usbPhyConfig = {
-	BOARD_USB_PHY_D_CAL, BOARD_USB_PHY_TXCAL45DP, BOARD_USB_PHY_TXCAL45DM,
+	BOARD_USB_PHY_D_CAL,
```

Comment:
you should avoid doing format changes in a file with the same commit where you do a functional change. Makes it harder to review. If you feel the need to do whitespace changes then please do them in a standalone commit.

Reply:
Updated